doc/Developing.md

```diff
@@ -0,0 +1,124 @@
+# Developing wdlTools
+
+## Setting up your development environment
+
+* Install JDK 8
+  * On mac with [homebrew]() installed:
+    ```
+    $ brew tap AdoptOpenJDK/openjdk
+    $ brew cask install adoptopenjdk8
+    # Use java_home to find the location of JAVA_HOME to set
+    $ /usr/libexec/java_home -V
+    $ export JAVA_HOME=/Library/Java/...
+    ```
+  * On Linux (assuming Ubuntu 16.04)
+    ```
+    $ sudo apt install openjdk-8-jre-headless
+    ```
+  * Scala will compile with JDK 11, but the JDK on DNAnexus worker instances is JDK 8 and will not be able to run a JAR file with classes compiled by a later version of Java
+* Install [sbt](https://www.scala-sbt.org/), which also installs Scala. Sbt is a make-like utility that works with the ```scala``` language.
+  * On MacOS: `brew install sbt`
+  * On Linux:
+    ```
+    $ wget www.scala-lang.org/files/archive/scala-2.12.1.deb
+    $ sudo dpkg -i scala-2.12.1.deb
+    $ echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
+    $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
+    $ sudo apt-get update
+    $ sudo apt-get install sbt
+    ```
+  * Running sbt for the first time takes several minutes, because it downloads all required packages.
+* We also recommend to install [Metals](https://scalameta.org/metals/), which enables better integration with your IDE
+  * For VSCode, install the "Scala (Metals)" and "Scala Syntax (official)" plugins
+* You will also need Python installed (we recommend version 3.6+) and dx-toolkit (`pip install dxpy`)
```

Comment:
I don't think this is needed here. Are we pulling in the dx-toolkit into this repository? I was hoping to keep it independent, smaller, and easier to test.

Reply:
You're right - I just missed it when I copied it over from dxWDL